documentation/docs.css

```diff
@@ -180,4 +197,65 @@ body {
     height: 100%;
     background-color: var(--bg-color);
   }
+
+  /* icon override */
+  .icon-badge-pos {
+    left: 1rem;
+    bottom: 0.8rem;
+  }
+
+  /* horizontal card */
+
+  .horizontal-btn-action {
+    display: none;
+  }
+
+  /* modal overrides */
+  .modal-container {
+    max-width: fit-content;
+  }
+
+  /* toast  */
+  #toast-container {
+    width: 20rem;
+  }
+
+  .toast-container {
+    position: fixed;
+    bottom: 4rem;
+    left: auto;
+  }
+
+  .navbar-icons {
+    display: none;
+  }
+
+  /* navigation css */
+
+  .nav-hamburger-icon {
+    display: flex;
+    flex-direction: column;
+    background-color: transparent;
+    border: none;
+    color: var(--black);
+    cursor: pointer;
+  }
+
+  .hamburger-wrapper {
+    margin-left: auto;
+    margin-right: 1rem;
+    display: flex;
+  }
+
+  .hamburger-list {
+    position: absolute;
+    top: 3.5rem;
+    right: 1rem;
+    z-index: 10;
+    outline: 1px solid var(--gray);
+    gap: 0.7rem;
+    background-color: var(--white);
+    cursor: pointer;
+    padding: 0.7rem 0.7rem;
```

Comment:
why two 0.7's?

Reply:
![nav-ex](https://user-images.githubusercontent.com/76784537/155009929-cac4613e-573f-4660-b6fb-94da97f05b0f.PNG)

so on click of that hamburger , that list opens and 0.7 looked better padding than 1 so used it.

Reply:
I mean `padding: 0.7rem;` is enough😅